axelar-chains-config/src/utils/verifyContract.js

```diff
@@ -12,7 +12,7 @@ const { writeFileSync } = require('fs');
  * @param {any[]} args
  * @returns {void}
  */
-const verifyContract = (env, chain, contract, args, options = {}) => {
+const verifyContract = async (env, chain, contract, args, options = {}) => {
```

Comment:
`async` switch requires adding an `await` everywhere this is used. Although, I'd prefer to handle this without having to make this method async.